sdcflows/workflows/fit/syn.py

```diff
@@ -214,9 +215,14 @@ def init_syn_sdc_wf(
     find_zooms = pe.Node(niu.Function(function=_adjust_zooms), name="find_zooms")
     zooms_epi = pe.Node(RegridToZooms(), name="zooms_epi")
 
+    syn_config = data.load(f"sd_syn{'_sloppy' * sloppy}.json")
```

Comment:
Question out of curiosity: why in the test we use `data.load.readable` and here `data.load`?

Reply:
`readable()` just makes sure that it has `.read_text()`, and doesn't guarantee that it's somewhere on the filesystem. I left this as `data.load()` because that's what it was before I moved it, and I didn't want to change it here.